unlock-js/package.json

```diff
@@ -12,14 +12,10 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "ethereumjs-util": "^6.0.0",
```

Comment:
I should know this is ethers providing a replacement for this? This is how we "compute" the address of the next lock.

Reply:
Not any more :). ethers provides a 1 liner for this. Also, it turns out ethereumjs-utils was a fork of an early version of ethers and is unmaintained so win-win

Reply:
Oh and tests cover that function fully FYI